Documentation/ApiOverview/Rte/InTheFrontend/Introduction.rst

```diff
@@ -0,0 +1,117 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Rich text editor; Frontend
+.. _rte-frontend-introduction:
+
+==================================================
+Including a Rich Text Editor (RTE) in the frontend
+==================================================
+
+When forms are added to a website it is sometimes desired to offer
+formatting options like bold, italic etc.. Rich Text Editors offer
+here extensive options that are configurable for the individual needs.
```

Comment:
```suggestion
When you add forms to a website you might want to offer
formatting options like bold, italic etc.. Rich Text Editors offer
extensive options that are configurable for individual needs.
```

Reply:
Thanks @flicstar , I must admit my text is not so smooth, I try to avoid the "you" usually, so it changes everything by introducing it again.
I read now already all your other comments and you make things nicer to read, Thanks!!!
About the "you" in the moment I feel a bit undecided, but don't know if there exists some styleguide about it.

Reply:
Good point, let me go check styling. I also need to check "a" or "an" RTE. 😁